packages/toolkit/src/apm/useApm.js

```diff
@@ -0,0 +1,14 @@
+import aragonPM from '@aragon/apm'
```

Comment:
Instead of have a specific function exposing each `apm.js` functionality needed we will expose the whole `apm.js` object.


In future iterations would like to have a similar behavior for `acl` and `kernel`.